identifiers/country-in/vidhan_sabha.csv

```diff
@@ -768,4 +768,3800 @@ ocd-division/country:in/state:mh/district:yt/cd:pusad,Pusad constituency; Yavatm
 ocd-division/country:in/state:mh/district:yt/cd:ralegaon,Ralegaon constituency; Yavatmal district; Maharashtra
 ocd-division/country:in/state:mh/district:yt/cd:umarkhed,Umarkhed constituency; Yavatmal district; Maharashtra
 ocd-division/country:in/state:mh/district:yt/cd:wani,Wani constituency; Yavatmal district; Maharashtra
-ocd-division/country:in/state:mh/district:yt/cd:yavatmal,Yavatmal constituency; Yavatmal district; Maharashtra
\ No newline at end of file
+ocd-division/country:in/state:mh/district:yt/cd:yavatmal,Yavatmal constituency; Yavatmal district; Maharashtra

```

Comment:
Can we sort thsi file so that each state's consituencies are grouped?

Reply:
Done